controller/controller.go

```diff
@@ -1235,6 +1235,7 @@ func (ctrl *ProvisionController) shouldProvision(ctx context.Context, claim *v1.
 // shouldDelete returns whether a volume should have its backing volume
 // deleted, i.e. whether a Delete is "desired"
 func (ctrl *ProvisionController) shouldDelete(ctx context.Context, volume *v1.PersistentVolume) bool {
+	klog.V(4).Infof("shouldDelete Volume : %+v", volume)
```

Comment:
Also, probably better to only log the name of the PV. Logging the whole object could get pretty verbose, especially with the managed fields.

Reply:
Ok changed. I was planning on just logging the whole object so I can look through the fields and figure out where it returned, but replaced that with return statements everywhere.